.github/workflows/release.yml

```diff
@@ -1,151 +1,157 @@
-        id: asset_2
-        run: |
-          helm package ./charts/csi-baremetal-deployment/ --version ${{ github.event.inputs.csi_operator_version }} --app-version ${{ steps.vars.outputs.app_version }}
-          echo "::set-output name=archive_name::csi-baremetal-deployment-${{ github.event.inputs.csi_operator_version }}.tgz"
-
-      - name: Upload csi-baremetal-deployment asset
-        uses: actions/upload-artifact@v2
-        with:
-          name: ${{ steps.asset_2.outputs.archive_name }}
-          path: ./${{ steps.asset_2.outputs.archive_name }}
-
-      - name: Generate changelog
-        id: changelog
-        uses: mikepenz/release-changelog-builder-action@v1
-        with:
-          token: ${{ secrets.CSI_WF_TOKEN }}
-          fromTag: ${{ steps.previous_tag.outputs.tag }}
-          toTag: ${{ github.event.inputs.release_tag }}
-          configuration: "./.github/workflows/changelog_config.json"
-
-      - name: Create release
-        uses: softprops/action-gh-release@v1
-        with:
-          token: ${{ secrets.GITHUB_TOKEN }}
-          tag_name: ${{ github.event.inputs.release_tag }}
-          name: Release ${{ github.event.inputs.release_tag }}
-          body: ${{ steps.changelog.outputs.changelog }}
-          draft: false
-          prerelease: true
-          files: |
-            ./${{ steps.asset_1.outputs.archive_name }}
-            ./${{ steps.asset_2.outputs.archive_name }}
-
-      - name: Checkout gh-pages
-        uses: actions/checkout@v2
-        with:
-          ref: gh-pages
-
-      - name: Download csi-baremetal-operator package
-        uses: actions/download-artifact@v2
-        with:
-          name: ${{ steps.asset_1.outputs.archive_name }}
-
-      - name: Download csi-baremetal-deployment package
-        uses: actions/download-artifact@v2
-        with:
-          name: ${{ steps.asset_2.outputs.archive_name }}
-
-      - name: Publish charts to gh-pages
-        run: |
-          echo `ls -Rl .`
-          assets_url=https://github.com/${{ github.repository }}/releases/download/${{ github.event.inputs.release_tag }}
-          helm repo index . --url ${assets_url} --merge docs/index.yaml
-          mv ./index.yaml -f ./docs
-
-      - name: Commit and push changes
-        id: commit_docs
-        uses: stefanzweifel/git-auto-commit-action@v4
-        with:
-          commit_message: "Update Helm repo index"
-          commit_options: --signoff
-          file_pattern: ./docs/index.yaml
-          branch: gh-pages
-
-      # - name: Send slack notification
-      #   uses: rtCamp/action-slack-notify@v2
-      #   env:
-      #     SLACK_WEBHOOK: ${{ secrets.SLACK_WEBHOOK }}
-      #     SLACK_CHANNEL: test-private-channel
-      #     SLACK_COLOR: ${{ job.status }}
-      #     SLACK_TITLE: 'Release ${{ github.event.inputs.release_tag }} is ready for review'
-      #     SLACK_MESSAGE: 'Please review this PR: ${{ steps.pr.outputs.pull-request-url }}'
+name: Release Charts

```

Comment:
Why diff shows that the whole file was changed?

Reply:
Idk, have the same question.